src/ui/popup_menu.c

```diff
@@ -116,6 +117,12 @@ ui_popup_item_menu (itemPtr item, guint button, guint32 activate_time)
 	g_menu_item_set_action_and_target (menu_item, "app.open-link-in-external-browser", "s", item_link);
 	g_menu_append_item (section, menu_item);
 
+    if(author){
+        g_menu_item_set_label (menu_item, _("Email The Author"));
+	    g_menu_item_set_action_and_target (menu_item, "app.email-the-author", "s", item_link);
```

Comment:
You are passing the item link as parameter to the action's callback, .maybe you intended to pass the author?  But then you are not actually using the parameter in your callback, as it acts on the selected item, and you also use the item's title. Either your action should take both strings as parameters, or the item's id, or nothing (preferably, I think it should use the item passed to this function).

Reply:
Clear.. I see now that these different ways of passing arguments are all used so will go with you suggestion and only pass the item->id.

Do you require a reply to all comments in this pull request?

Reply:
No, actual replies are not necessary. Fixing the issues in a commit is enought.